cmd/janitor/main.go

```diff
@@ -83,6 +83,9 @@ func run(cmd *cobra.Command, args []string) {
 	if cfg.ClairClient != nil {
 		go tasks.GoQueuedJobLoop(ctx, 3, janitor.CheckVulnerabilitiesForNextManifest())
 	}
+	if cfg.Trivy != nil {
+		go janitor.CheckTrivySecurityStatus(nil).Run(ctx)
```

Comment:
We should at least make a note here that this needs to be scaled to multiple goroutines eventually.